build.gradle

```diff
@@ -500,6 +500,14 @@ project('spring-integration-mail') {
 	[compileJava,compileTestJava]*.options*.compilerArgs = ["${xLintArg},-path"]
 }
 
+project('spring-integration-mock') {
```

Comment:
Yes, that's fine, we just cannot have the same package in 2 jars but, yes, I agree it's less invasive to leave the "old" classes in their original packages.